src/Explorer/Panes/AddCollectionPanel.tsx

```diff
@@ -1164,11 +1269,16 @@ export class AddCollectionPanel extends React.Component<AddCollectionPanelProps,
     }
 
     const uniqueKeyPolicy: DataModels.UniqueKeyPolicy = this.parseUniqueKeys();
-    const partitionKeyVersion = this.state.useHashV2 ? 2 : undefined;
+    const partitionKeyVersion = this.state.useHashV1 ? undefined : 2;
     const partitionKey: DataModels.PartitionKey = partitionKeyString
       ? {
-          paths: [partitionKeyString],
-          kind: "Hash",
+          paths: [
+            partitionKeyString,
```

Comment:
Do we need to include the `partitionKeyString` in the `paths` even when sub partitionkey is enabled?

Reply:
yeah, because subpartition keys are stored in separate array so need to concatenate